.pipelines/js-release.yml

```diff
@@ -26,37 +26,21 @@ parameters:
   default: true
 
 steps:
-- task: NodeTool@0
-  name: NodeTool1
-  displayName: Use Node 14.x
+- task: UseNode@1
+  displayName: Use Node 20.x
   inputs:
-    versionSpec: 14.x
+    versionSpec: '20.x'
 
 - bash: |
-   npm install -g npm@8.11.0
-  workingDirectory: source/nodejs
-  displayName: 'Bash - npm install -g npm 8.11.0'
-
-- bash: |
-   npm ci 
+   npm ci --maxsockets=1
   workingDirectory: source/nodejs
   displayName: 'Bash - npm ci'
 
-- bash: |
-   npx lerna bootstrap --ci
```

Comment:
We don't need to bootstrap anymore?

Reply:
Oh I see, we are using npm now isntead of lerna?

Reply:
that's correct -- I updated the README, but I should've noted that in the description above as well... I'll update that now. :)